src/main/java/backend/UpdateController.java

```diff
@@ -107,32 +111,37 @@ public void processAndRefresh(List<FilterPanel> filterPanels) {
     }
 
     /**
-     * Filters, sorts and counts issues within the model according to the given filter expressions.
+     * Filters, sorts and counts issues within the model according to the given panels' filter expressions.
      * In here, "processed" is equivalent to "filtered, sorted and counted".
      *
-     * @param filterExprs Filter expressions to process.
+     * @param filterPanels Filter panels to process.
      * @return Filter expressions and their corresponding issues after filtering, sorting and counting.
      */
-    private Map<FilterExpression, List<GuiElement>> processFilter(List<FilterExpression> filterExprs) {
+    private Map<FilterExpression, List<GuiElement>> processPanels(List<FilterPanel> filterPanels) {
```

Comment:
I would disagree with renaming it to `panels`. Logic doesn't have to care about what is holding the filters; it just has to process the filters themselves. This is why there is a `GUIController`, which distributes the filters to their respective panels.